CAP/gap/Finalize.gi

```diff
@@ -194,6 +194,8 @@ InstallMethod( IsFinalized,
         derivation_list := derivation_list{ Difference( [ 1 .. Length( derivation_list ) ], current_installs ) };
         
     od;
+
+    RECALCULATE_ALL_METHOD_RANKS();
```

Comment:
if CompareVersionNumbers( GAPInfo.Version, "4.10.0" ) then
    ValueGlobal( "RECALCULATE_ALL_METHOD_RANKS" )();
fi;

Reply:
Currently, this patch is only valid for master/4.11

Reply:
Then
```
if IsBoundGlobal( RECALCULATE_ALL_METHOD_RANKS ) then
    ValueGlobal( "RECALCULATE_ALL_METHOD_RANKS" )();
fi;
```